src/components/WebexVideoSettings/WebexVideoSettings.jsx

```diff
@@ -16,15 +18,23 @@ import WebexMeetingControl from '../WebexMeetingControl/WebexMeetingControl';
  */
 export default function WebexVideoSettings({meetingID, className, style}) {
   const cssClasses = webexComponentClasses('video-settings', className);
+  const [, display] = useMeetingControl('switch-camera', meetingID);
 
   return (
     <div className={cssClasses} style={style}>
-      <Title>Camera</Title>
-      <WebexMeetingControl type="switch-camera" meetingID={meetingID} />
-      <div className="media">
-        <WebexLocalMedia banner="Preview" mediaType="video" meetingID={meetingID} />
-        <div className="preview"><h3>Preview</h3></div>
-      </div>
+      {display.options?.length > 0
```

Comment:
let's use the condition `display.options?.length === 0`, because only when the options array exists but it's empty, the meaning is "there is no camera available". While options are loading, for example, display.options is null, and we don't want to display the "no media" error in that case, even for a fraction of a second.